src/steps/02-EvaluationCriteria/JandA/MinimumRequirements.vue

```diff
@@ -1,25 +1,188 @@
 <template>
-  <v-form ref="form" lazy-validation>
-    <v-container fluid class="container-max-width">
-      <v-row>
-        <v-col class="col-12">
-          <h1 class="mb-3">
-            Tell us about your minimum government requirements
-          </h1>
-          <p>
-            Intro paragraph
-          </p>
-        </v-col>
-      </v-row>
-    </v-container>    
-  </v-form>
+  <div>
+    <v-form ref="form" lazy-validation>
+      <v-container fluid class="container-max-width">
+        <v-row>
+          <v-col class="col-12">
+            <h1 class="mb-3">
+              Tell us about your minimum government requirements
+            </h1>
+            <p>
+              In the field below, explain what is being supported and provide any
+              information regarding your verified minimum requirements. Discuss, if
+              applicable, how the required delivery/performance date impacted the
+              decision to restrict competition. You can fill in the blank to complete
+              the suggested sentences or write your own explanation.
+            </p>
+            <ATATTextArea
+                id="minGovReqExplanation"
+                class="max-width-740"
+                :rows="7"
```

Comment:
make rows 11 to match height in Figma more closely
![Screen Shot 2023-04-25 at 3 27 21 PM](https://user-images.githubusercontent.com/90333349/234382630-39f97cf0-63d8-42bb-81a5-29c02f660645.png)